src/ir_Midea.cpp

```diff
@@ -231,7 +231,7 @@ void IRMideaAC::setMode(const uint8_t mode) {
       _.Mode = mode;
       break;
     default:
-      _.Mode = kMideaACAuto;
+      this->setMode(kMideaACAuto);
```

Comment:
Ditto.

Reply:
should I keep "this->" or not?

Reply:
I have no strong viewpoint on `this->` one way or the other. So, your choice.
@NiKiZe any opinions?

Reply:
no strong opinion, but I think we are mostly without it (and have removed it in some places already) so are leaning towards removal being preferred.

Reply:
Agreed.